phoenix-core/src/main/java/org/apache/phoenix/hbase/index/Indexer.java

```diff
@@ -562,7 +614,9 @@ public void postBatchMutateIndispensably(ObserverContext<RegionCoprocessorEnviro
           this.builder.batchCompleted(miniBatchOp);
 
           if (success) { // if miniBatchOp was successfully written, write index updates
```

Comment:
nit: I believe this comment is now incorrect

Reply:
Yes, you are right. I will update it